### ISSUE TITLE
Figure title is not translated to lt-lt in PDF

### ISSUE THREAD

**Issue by asesulina** [dita-ot/dita-ot#2254]
Please review attached image. Language that I am using is Lithuanian (lt-lt), when publishing content to PDF2, Figure title is not translated. HTML2 translates figure title correctly. xml:lang="lt-lt" is set on all pages.

Here is the code for image and title:
```
 <itemgroup>
     <fig id="fig2">
      <title>ATutor pagrindinis dėstytojo langas</title>
      <image placement="break" 
       href="../image/2pav.jpg" align="center" 
       id="pav2">
       <alt>ATutor pagrindinis dėstytojo langas</alt>
      </image>
     </fig>
    </itemgroup>
```
![pdfnottranslatedfiguretitle](https://cloud.githubusercontent.com/assets/17761749/13666344/bf4251de-e6b9-11e5-9544-4734e5ef0476.jpg)

**Comment by xephon2** [comment]
Which DITA-OT version are you using?

**Comment by asesulina** [comment]
dita-ot-2.2.1

**Comment by xephon2** [comment]
Lithuanian is not supported by the PDF2 plugin, see [Supported languages: PDF transformations](http://www.dita-ot.org/2.2/user-guide/DITA-globalization-pdf.html). But it is quite easy to implement. Add a new `<lang>` element in the `~\plugins\org.dita.pdf2\cfg\common\vars\strings.xml` file:

`<lang filename="lt.xml" xml:lang="lt"/>`

Copy the `~\plugins\org.dita.pdf2\cfg\common\vars\en.xml` file to `~\plugins\org.dita.pdf2\cfg\common\vars\lt.xml` and translate it.

The whole process is also explained in [Modifying or adding generated text](http://www.dita-ot.org/dev/dev_ref/plugin-addgeneratedtext.html).

You are very welcome to return your translations to become part of the DITA-OT.

**Comment by xephon2** [comment]
Ah, I see that Lithuanian has just been merged, see #2137. So you just can download the [lt.xml](https://raw.githubusercontent.com/robander/dita-ot/584a434c139aa1fcaa21241da34d33bbe3a688b1/src/main/plugins/org.dita.pdf2/cfg/common/vars/lt.xml)

**Comment by xephon2** [comment]
I missed something. You have to create/modify some other files (because it is a new language and you do not want to create a PDF plugin):

- `src/main/plugins/org.dita.pdf2/cfg/catalog_template.xml`
- `src/main/plugins/org.dita.pdf2/cfg/common/index/lt.xml`
- `src/main/plugins/org.dita.pdf2/cfg/common/vars/lt.xml`
- `src/main/plugins/org.dita.pdf2/cfg/common/vars/strings.xml`
- `src/main/plugins/org.dita.pdf2/cfg/fo/i18n/lt.xml`

You have to exactly repeat the changes of this commit: [584a434](https://github.com/robander/dita-ot/commit/584a434c139aa1fcaa21241da34d33bbe3a688b1)
